convert2rhel/pkghandler.py

```diff
@@ -935,6 +944,10 @@ def _get_packages_to_update_dnf(reposdir):
     :type reposdir: str | None
     """
     packages = []
+    # Disable the rhel repositories when checking if the packages are up to date.
+    # When user specifies rhel repos to use during conversion, disabling them is also needed.
+    # Note: Do not depend on the --no-rhsm option, enable repos can be specified without it.
+    disable_repos = ["rhel*"] + tool_opts.enablerepo
```

Comment:
One tip, that may not be 100% necessary here (and definitely not a blocker), but you could move the disable_repos to be a parameter in the function, this way you only have to identify the rhel repos + tool_opts.enablerepo once in the caller, and pass this down to the functions.

Reply:
Yeah, it would help with decreasing duplicity of code. I'll do it, I like the idea. And as I'm looking on it, I see I forgot to disable the `tool_opts.enablerepo` on other places.